### PR TITLE
[berte-547] GitHub action status error

### PR DESCRIPTION
**Pull request by erwan-b**
Change api call from checksuites to checkruns
- Checksuites got others app, like dependa bot, infos
- CheckRuns get only github actions infos

### CODE REVIEW

**Review by tcarmet — APPROVED** [review]

File: CHANGELOG [hunk 1]
```diff
@@ -1,12 +1,14 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## [3.6.0] - 2021-05-26
```
Comment (tcarmet) [inline CHANGELOG:4]:
```suggestion
## [3.6.1] - 2021-05-26
```

**Review by bertranddemiddelaer — APPROVED** [review]

**Review by cmeron-scality — APPROVED** [review]
blind approve :)

**Pull request merged**